target/linux/rockchip/files-5.10/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
```

Comment:
This target still uses patch to add devices' dts, please keep consistent.

Reply:
This is a hack made by upstream, in my option it's fine to keep it as-is.